packages/data-point/lib/entity-types/base-entity/resolve.test.js

```diff
@@ -197,6 +197,49 @@ describe('ResolveEntity.resolveEntity', () => {
         expect(err).toHaveProperty('message', 'test')
       })
   })
+
+  test('inputType -throws error if inputType does not pass', () => {
```

Comment:
There's an extra `-` in here.

Oh, I see, it's used to separate the text. Can we put all the `inputType` tests together then use `It should` to describe each? Same for `outputType`